keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -660,8 +661,12 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - The Feature is implemented behind `WatchList` feature flag
 - Initial e2e tests completed and enabled
 - Scalability/Performance tests confirm gains of this feature
+- Add support for watchlist to APF
+
+#### Beta
```

Comment:
In Consistent Read from Cache KEP we added the mechanim for manual progress notify:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache#use-requestprogress-to-enable-automatic-watch-updates

Given that this code is shared across both features, we should have the answer for it [or somehow disable that for now and rely purely on periodic ones]
xref: https://github.com/kubernetes/enhancements/pull/4134/files

In any case - we should decide on something and mention it here in the KEP.